airbyte-integrations/connectors/source-stripe/integration_tests/expected_subscriptions_statuses_records.json

```diff
@@ -0,0 +1,19 @@
hod": "charge_automatically", "created": 1594766545, "current_period_end": 1627819200, "current_period_start": 1625140800, "customer": "cus_HeFKZGFvcdVAAd", "days_until_due": null, "default_payment_method": null, "default_source": null, "default_tax_rates": [], "discount": null, "ended_at": null, "items": {"object": "list", "data": [{"id": "si_HeFKnht1gZEPtM", "object": "subscription_item", "billing_thresholds": null, "created": 1594766546, "metadata": {}, "plan": {"id": "plan_HDfijky2JcM0pm", "object": "plan", "active": true, "aggregate_usage": "sum", "amount": null, "amount_decimal": null, "billing_scheme": "tiered", "created": 1588637439, "currency": "usd", "interval": "month", "interval_count": 1, "livemode": false, "metadata": {}, "nickname": "free-overage", "product": "prod_H07MQuAFARZ6K6", "tiers": [{"flat_amount": null, "flat_amount_decimal": null, "unit_amount": 0, "unit_amount_decimal": "0", "up_to": 5000}, {"flat_amount": null, "flat_amount_decimal": null, "unit_amount": 0, "unit_amount_decimal": "0", "up_to": null}], "tiers_mode": "graduated", "transform_usage": null, "trial_period_days": null, "usage_type": "metered"}, "price": {"id": "plan_HDfijky2JcM0pm", "object": "price", "active": true, "billing_scheme": "tiered", "created": 1588637439, "currency": "usd", "livemode": false, "lookup_key": null, "metadata": {}, "nickname": "free-overage", "product": "prod_H07MQuAFARZ6K6", "recurring": {"aggregate_usage": "sum", "interval": "month", "interval_count": 1, "trial_period_days": null, "usage_type": "metered"}, "tiers_mode": "graduated", "transform_quantity": null, "type": "recurring", "unit_amount": null, "unit_amount_decimal": null}, "subscription": "sub_HeFKfIvdgFh3qw", "tax_rates": []}], "has_more": false, "total_count": 1, "url": "/v1/subscription_items?subscription=sub_HeFKfIvdgFh3qw"}, "latest_invoice": "in_1J8OZSIEn5WyEQxnqy1SOqzg", "livemode": false, "metadata": {"eligibleForTrial": "false"}, "next_pending_invoice_item_invoice": null, "pause_collection": null, "pending_invoice_item_interval": null, "pending_setup_intent": null, "pending_update": null, "plan": {"id": "plan_HDfijky2JcM0pm", "object": "plan", "active": true, "aggregate_usage": "sum", "amount": null, "amount_decimal": null, "billing_scheme": "tiered", "created": 1588637439, "currency": "usd", "interval": "month", "interval_count": 1, "livemode": false, "metadata": {}, "nickname": "free-overage", "product": "prod_H07MQuAFARZ6K6", "tiers": [{"flat_amount": null, "flat_amount_decimal": null, "unit_amount": 0, "unit_amount_decimal": "0", "up_to": 5000}, {"flat_amount": null, "flat_amount_decimal": null, "unit_amount": 0, "unit_amount_decimal": "0", "up_to": null}], "tiers_mode": "graduated", "transform_usage": null, "trial_period_days": null, "usage_type": "metered"}, "quantity": 1, "schedule": null, "start_date": 1594766545, "status": "active", "tax_percent": null, "transfer_data": null, "trial_end": 1596283200, "trial_start": 1595483518}, "emitted_at": 1626172757000}}
```

Comment:
where is this used?

Reply:
This subset of records is used in `source-acceptance-tests` for checking for expected_records in basic_read for subscription stream.